.github/stale.yml

```diff
@@ -0,0 +1,58 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 700  # start with a large number and reduce shortly
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7 # start with a large number and reduce over time
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - pinned
+  - security
+  - "[Status] Maybe Later"
+
```

Comment:
Add "feature_request" and "API_design" (not sure of syntax)? Those seem to hang around for a while.

Reply:
I'll merge if you're OK with this @dcherian , unless there are any other comments?

Reply:
Fine by me. Sorry for the delay in response.